services/parse/auth/index.js

```diff
@@ -43,27 +45,27 @@ function retrieveSignOutFunction() {
 }
 
 function retrieveForgotPasswordFunction(params) {
-  Parse.Cloud.run('forgotPassword', params).then((result) => result);
+  Parse.Cloud.run('forgotPassword', params).then((result) => console.log(result));
 }
 
 function retrieveCurrentUserFunction() {
-  const u = Parse.User.currentUser();
+  const u = Parse.User.current();
   if (u) {
     const user = new Parse.User();
     user.id = u.id;
     user.name = u.get('username');
     user.email = u.get('email');
     user.organization = u.get('organization');
     user.role = u.get('role');
-    // console.log(user);
+    console.log(user);
     return user;
   }
-  // console.log(null);
+  console.log(null);
   return null;
 }
 
 function retrieveDeleteUserFunction(params) {
-  Parse.Cloud.run('deleteUser', params).then((result) => result);
+  Parse.Cloud.run('deleteUser', params).then((result) => cosole.log(result));
```

Comment:
You'll have to remove the console.log regardless, but spelling for this one.


@jamccomb92 did you run lint before making this pull request?